contrib/graphite/README.md

```diff
@@ -0,0 +1,46 @@
+Blueflood Finder
+================
+
+The BF finder is the graphite plugin that allows graphite and grafana to use blueflood as a backend.
+
+### Setup
+
+Get the [blueflood](https://github.com/rackerlabs/blueflood) repo from github. Execute the following commands
+
+    cd $BLUEFLOOD_REPO_LOCATION/contrib/graphite
+    virtualenv bf-finder
+    source bf-finder/bin/activate
+    pip install .
+ 
+## Development environment on vagrant
+
+We can setup grafana server on a vagrant vm which visualizes local blueflood using BF finder. To spin up 
+vagrant vm use the below commands.
+
+    vagrant box add ubuntu/trusty64
+    vagrant up
```

Comment:
Yes. Ideally this contrib/graphite should be in its own project. We should talk about it and do it as part of the next story which is about creating a pypi package.